buf.gen.deps.js.yaml

```diff
@@ -1,5 +1,19 @@
 version: v1
+managed:
+  enabled: true
+  java_multiple_files: true
+  java_package_prefix:
+    default: ai.visma
+    except:
+      - buf.build/googleapis/googleapis
+      - buf.build/grpc-ecosystem/grpc-gateway
+      - buf.build/validate/validate
 plugins:
   - remote: buf.build/protocolbuffers/plugins/js
     out: gen/js
     opt: import_style=commonjs,binary
+  - remote: buf.build/protocolbuffers/plugins/java
+    out: gen/java
+  - remote: buf.build/protocolbuffers/plugins/csharp
```

Comment:
I see why this is needed but's confusing that this file named `...js.yaml` also generated dependencies for Java and C#. I think it's better to rename this file to `buf.gen.deps.yaml` now.

Reply:
If you do this, also change the comment [in the Makefile](https://github.com/e-conomic/vmlapis/blob/61426027740ed7dbc56ecae31bb452a2c8d08c8f/Makefile#L61).

Reply:
I'm investigating now if this is even needed for java and c#, since we're using [this option](https://github.com/e-conomic/vmlapis/blob/bf4e7bfc0cebd68e16b563b9ef16d2d0d61a3ce4/Makefile#L16)

Reply:
Conclusion: yes, yes it is :D